infra/registry/output.tf

```diff
@@ -0,0 +1,12 @@
+resource "null_resource" "write_hosts_file" {
+  triggers = {
+    ip_addresses = sha256(jsonencode(openstack_networking_floatingip_v2.fip[*].address))
+  }
+
+  provisioner "local-exec" {
+    command = <<-EOT
+      echo "${join("\n", openstack_networking_floatingip_v2.fip[*].address)}" > ./docker-registry/hosts
+      echo "[defaults]\ninventory = ./hosts\nremote_user = ubuntu\nhost_key_checking = false" > ./docker-registry/ansible.cfg
```

Comment:
`terraform apply` won't be ran from the `registry` directory. It will be ran from the parent directory `mekstack/infra`. Set up absolute path with terraform `path` variable.